e2e-env/docker-compose.yml

```diff
@@ -18,8 +18,6 @@ services:
     volumes:
       - ..:/var/www/html/modules/ps_eventbus:rw
       - ./init-scripts:/tmp/init-scripts:ro
-      - /var/www/html/modules/ps_eventbus/vendor
```

Comment:
If you remove this, live code won't work on platforms with working permissions

Reply:
But the module folder is accessible in :rw. It is therefore not necessary to indicate the vendor folder in addition.

Furthermore, leaving this parameter makes the installation of the module impossible, because it considers the folder as empty (symbolic link)